src/sentry/routers.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import absolute_import
+
+
+def db_for_model(model):
+    db_table = model._meta.db_table
+    if db_table in {
```

Comment:
If you define this set at the module level you'll only create it once. Right now the set is built on each method call.

Reply:
Initially, that was meant as a throw-away code but if we decide to have multiple DBs also in sentry dev/ci I will change it as suggested.